### PR TITLE
Review Trial code

### PR DESCRIPTION
**Pull request by YKerroum**
### Give a review of my trial project code:

- [x] Create an empty repo.
- [x] Add a copy of my trial code. 
- [ ] Give it a review.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,2 @@
-# Review-your-project-from-trials-
+# microverse-triel
+microverse
```
Comment (YKerroum) [inline README.md:2]: I think that you should use the [Microverse README template](https://github.com/microverseinc/readme-template/blob/master/README.md) to have a professional README file.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    
+    <section id="welcome-section">
+        <h1>Antony And Youssef</h1>
+    </section>
+
+    <section id="projects">
+        <a href="#" class="project-tile"><img src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute.jpg" alt=""></a>
+    </section>
+
+    <section id="">
```
Comment (YKerroum) [inline index.html:28]: Please provide the section ID a name; in accordance with [HTML & CSS best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md), ID tags must have names.

**Review by YKerroum — COMMENTED** [review]
Changes required ♻️ 

- [ ] Please read the following comments.
- [ ] Make the necessary modifications.
- [ ] Launch a review quest. 
**Thanks 😃 **

**Review by ShahierNashaat — APPROVED** [review]
Hi @YKerroum , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Approved :ok_hand: :ok_hand: :ok_hand:

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**